### PR TITLE
Split socketserver and asyncio configs

### PR DESCRIPTION
**Pull request by vintozver**
Add socketserver IPv6 implementation
Add standard network constants

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,10 +21,10 @@ Usage
 .. code:: python
 
     import asyncio
-    import ssdp
+    import ssdp.asyncio
 
 
-    class MyProtocol(ssdp.SimpleServiceDiscoveryProtocol):
+    class MyProtocol(ssdp.asyncio.SimpleServiceDiscoveryProtocol):
```
[the remark this answers is not in the document]
Comment (vintozver) [reply README.rst:27]: yep, I specifically thought about that - cause I made the lib not just "asyncio" but also "socketserver".

I will move objects to the objects module and top level one will be the compat layer. So we will be good for the users' prospective.

File: ssdp/__init__.py [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+
+__all__ = ('NetworkConstants', 'SSDPMessage', 'SSDPRequest', 'SSDPResponse')
 
-__all__ = ('SSDPRequest', 'SSDPResponse', 'SimpleServiceDiscoveryProtocol')
 
 logger = logging.getLogger('ssdp')
 
 
-class SSDPMessage:
+class NetworkConstants(object):
```
[the remark this answers is not in the document]
Comment (vintozver) [reply ssdp/__init__.py]: moved to a separate module

**Comment by vintozver** [comment]
I verified on my local changes of iot-upnp

image

**Comment by codingjoe** [comment]
Thanks @vintozver for rebasing and addressing my comments. Interesting, might I ask what you are building with the Raspberry Pi and this package. I am always curious, what people are building.
Would you also mind addressing the check suite failures? Thanks, Joe

**Comment by codingjoe** [comment]
@vintozver I know the test CI suite is rather strict. If you need a hand, I am happy to help. Just let me know, I don't want to push to your branch without your consent.

**Comment by codingjoe** [comment]
@vintozver I'd love to get this change life. I rebased it and fixed a couple of small things. However, there are some places where I'd need your help. Mainly test coverage and documentation. Preferably, every object and method has a doc string. Including the network addresses.

Would you be interested to push this over the finish line? Cheers, Joe

**Comment by codingjoe** [comment]
Hi @vintozver. I just released version 1.2 which includes a larger refactoring inspired by your pull-request. It doesn't include a full server implementation yet, I have a draft for that. However, I will close this PR, since it's redundant. Cheers! Joe

**Pull request closed without merging**